launch/tier4_control_launch/launch/control.launch.xml

```diff
@@ -192,7 +192,6 @@
         <remap from="~/input/pointcloud" to="/perception/obstacle_segmentation/pointcloud"/>
         <remap from="~/input/velocity" to="/vehicle/status/velocity_status"/>
         <remap from="~/input/imu" to="/sensing/imu/imu_data"/>
-        <remap from="~/input/odometry" to="/localization/kinematic_state"/>
```

Comment:
@danielsanchezaran
This change is intended?

Reply:
@kyoichi-sugahara yes, that topic is not used